frontends/web/src/components/icon/icon.tsx

```diff
@@ -33,6 +33,7 @@ import closeXWhiteSVG from './assets/icons/close-x-white.svg';
 import closeXDarkSVG from './assets/icons/close-x-dark.svg';
 import guideSVG from './assets/icons/guide.svg';
 import menuSVG from './assets/icons/menu.svg';
```

Comment:
we could remove this unused `menuSVG` import.

Reply:
removed now